src/seq.rs

```diff
@@ -525,6 +558,94 @@ impl PortSubscribe {
 
 }
 
+/// [snd_seq_query_subs_type_t](https://www.alsa-project.org/alsa-doc/alsa-lib/group___seq_subscribe.html) wrapper
+#[derive(Copy, Clone)]
+pub enum SubscriptionQueryType {
+    READ = alsa::SND_SEQ_QUERY_SUBS_READ as isize,
+    WRITE = alsa::SND_SEQ_QUERY_SUBS_WRITE as isize,
+}
+
+/// [snd_seq_query_subscribe_t](https://www.alsa-project.org/alsa-doc/alsa-lib/group___seq_subscribe.html) wrapper
+//(kept private, functionality exposed by PortSubscribeIter)
+struct SubscriptionQueryInfo(*mut alsa::snd_seq_query_subscribe_t);
```

Comment:
Rename to "QuerySubscribe"

Reply:
Also I'm not sure if there is a reason not to make it public?

Reply:
Renamed.

The reason I kept it private is because this struct is used internally by PortSubscribeIter, but would have no real purpose to users of this crate (they could create it and get/set values but would do not useful work).  That being said if you prefer it public let me know, I'll change it.